jekyll/install.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: Install
```

Comment:
Why change this? Having a page with a filename install.html and a title that's "Download" will be confusing.

Reply:
This needs to be changed for highlighting in the top bar to work correctly for all pages.